### ISSUE TITLE
Remove ShibbolethSSO and SAML 1

### ISSUE THREAD

**Issue by nynymike** [GluuFederation/oxShibboleth#21]
Remove from SAML metadata:

```
<ArtifactResolutionService Binding="urn:oasis:names:tc:SAML:1.0:bindings:SOAP-binding" Location="https://goblin.gluu.info:9443/idp/profile/SAML1/SOAP/ArtifactResolution" index="1"/>

<AttributeService Binding="urn:oasis:names:tc:SAML:1.0:bindings:SOAP-binding" Location="https://goblin.gluu.info:9443/idp/profile/SAML1/SOAP/AttributeQuery"/>

<SingleSignOnService Binding="urn:mace:shibboleth:1.0:profiles:AuthnRequest" Location="https://goblin.gluu.info/idp/profile/Shibboleth/SSO"/>
```

Also

```
<AttributeAuthorityDescriptor protocolSupportEnumeration="urn:mace:shibboleth:1.0 urn:oasis:names:tc:SAML:1.1:protocol urn:oasis:names:tc:SAML:2.0:protocol">
```

**Comment by dmogn** [comment]
https://github.com/GluuFederation/oxShibboleth/issues/11

**Comment by uboznikov** [comment]
[- Fix for #465](https://github.com/GluuFederation/oxTrust/commit/ac46049fa413c4caa2ae3a3f2353b137f1540c9e)

**Comment by zamilskhan** [comment]
Fix confirmed
